backend/app/minio/config.py

```
@@ -14,8 +13,13 @@
-    print(f"Bucket '{BUCKET_NAME}' created successfully.")
-else:
-    print(f"Bucket '{BUCKET_NAME}' already exists.")
+def initialize_minio_bucket():
+    
+    try:
+        if not MINIO_CLIENT.bucket_exists(BUCKET_NAME):
+            MINIO_CLIENT.make_bucket(BUCKET_NAME)
+            print(f"Bucket '{BUCKET_NAME}' created successfully.")
+        else:
+            print(f"Bucket '{BUCKET_NAME}' already exists.")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)